apps/ex_nvr_web/lib/ex_nvr_web/live/recordings_list_live.ex

```diff
@@ -146,6 +149,15 @@ defmodule ExNVRWeb.RecordingListLive do
      )}
   end
 
+  @impl true
+  def handle_info(_msg, socket) do
+    params =
+      Map.merge(socket.assigns.filter_params, socket.assigns.pagination_params)
+      |> Map.merge(socket.assigns.sort_params)
+
+    {:noreply, push_patch(socket, to: Routes.recording_list_path(socket, :list, params))}
```

Comment:
This code is the same as the one on `handle_event("popu", ...`. Consider creating a new function.

Reply:
Actually it's not the same, even if we use construct the params the same way, in the close-popup event, we use `push_navigate` and here we use `push_patch`

Reply:
Ok, however this is still has an issue. If you're watching a video in the popup and this even occurs, the popup will close. I think we should avoid `push_navigate` and execute the same code as `handle_params`.